parser/test/test_parse_qop.c

```diff
@@ -65,10 +65,12 @@ void test_parse_qop_val(void)
 	}
 }
 
-void test_parse_qop_oob(const str *tstr, void *farg)
+void test_parse_qop_oob(const str *tstr, enum oob_position where, void *farg)
 {
 	struct authenticate_body *ap = farg;
 
 	parse_qop_value(*tstr, ap);
-	ok(1, "oob check: parse_qop_value(\"%.*s\")", tstr->len, tstr->s);
+	ok(1, "oob check: parse_qop_value(%s\"%.*s\"%s)",
+	    where == OOB_PRE ? "->" : "", tstr->len, tstr->s,
+	    where == OOB_POST ? "<-" : "");
```

Comment:
Code smell:

These 3 lines are copy-paste from `test_parse_authenticate_body_oob()`.  Depending on how this code is going to evolve, maybe this `"->"` and `"<-"` printing code will give more value/reusage if placed in `test_oob()`, with some parameter refactoring, if needed.